release-tools/cloudformation/plugin-iam-users.yml

```diff
@@ -0,0 +1,177 @@
+Description: Template for setting up IAM users for each plugin repository so they can publish artifacts into the staging bucket
+Parameters:
+  BucketName:
+    Type: String
+    Description: The name of the staging bucket
+    Default: staging.artifacts.opendistroforelasticsearch.amazon.com
+Resources:
+  Alerting:
+    Type: "AWS::IAM::User"
+    Properties:
+      UserName: Alerting
+      Policies:
+        - PolicyName: S3_Write_Access
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              Effect: Allow
+              Action: "s3:PutObject"
+              Resource:
+                - !Sub "arn:aws:s3:::${BucketName}/snapshot/elasticsearch-plugins/alerting/*"
+  AlertingKibana:
+    Type: "AWS::IAM::User"
+    Properties:
+      UserName: AlertingKibana
+      Policies:
+        - PolicyName: S3_Write_Access
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              Effect: Allow
+              Action: "s3:PutObject"
+              Resource:
+                - !Sub "arn:aws:s3:::${BucketName}/snapshot/kibana-plugins/alerting/*"
+  SQL:
+    Type: "AWS::IAM::User"
+    Properties:
+      UserName: SQL
+      Policies:
+        - PolicyName: S3_Write_Access
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              Effect: Allow
+              Action: "s3:PutObject"
+              Resource:
+                - !Sub "arn:aws:s3:::${BucketName}/snapshot/elasticsearch-plugins/sql/*"
+                - !Sub "arn:aws:s3:::${BucketName}/snapshot/clients/sqlcli/*"
```

Comment:
would also need to `kibana-plugins/sql-workbench/*` access

Reply:
Added